### PR TITLE
Renew access token fixes + Log verbosity.

### PR DESCRIPTION
**Pull request by dmxfee**
-Renew access token fixes.
-Added log verbosity functionality.

### CODE REVIEW

File: changelog.txt [hunk 1]
```diff
@@ -1,3 +1,14 @@
+Version 1.0.7
+==========================
```
Comment (weijjia) [inline changelog.txt]: have you already taken our latest change?

**Comment by weijjia** [comment]
What is the problem when you renew token and why do you want to make the change?
Also, could you please send the pull request to dev branch?

File: changelog.txt (hunk 1 above)
Comment (dmxfee) [inline changelog.txt]: Yes, this pull request included the changes for version 1.0.7

**Comment by weijjia** [comment]
Please send a pull request to dev.

**Comment by dmxfee** [comment]
Closing this pull request and making a new one on the dev branch as requested.

**Pull request closed without merging**